content/open-source/accoutrement.rst

```diff
@@ -0,0 +1,96 @@
+public: yes
```

Comment:
Would it be possible to continue using `/accoutrement/`, with the new content? Or is it important to nest it under `/open-source/`?